lib/pacemaker/pcmk_simulate.c

```diff
@@ -512,3 +512,27 @@ pcmk__simulate(pe_working_set_t *data_set, pcmk__output_t *out, pcmk_injections_
 
     return rc;
 }
+
+#ifdef BUILD_PUBLIC_LIBPACEMAKER
```

Comment:
We don't need to guard this function. Only the stonith APIs are guarded b/c we don't want to expose stonith_t in the high-level API. (I was hoping we could get to replacing stonith_t with pcmk_ipc_api_t, but that's probably a long ways off -- it might be sufficient to make the stonith_t connection inside the high-level APIs rather than pass it to them.)

Reply:
Ah, for some reason I thought the define was hiding all these function until we'd had long enough to think about them and decide they were good enoguh to expose as public API.  I guess we'd better think more about whether these are correct, then.